CHANGELOG.md

```diff
@@ -94,6 +103,8 @@
 - Fixed `Get-WSAServerDetails` cmdlet where incorrect initial check was looking for VMware Aria Automation not VMware Aria Suite Lifecycle.
 - Fixed `Add-vRLIAuthenticationWSA` cmdlet description and synopsis which was incorrect.
 - Fixed `Add-NsxtLdapRole` cmdlet which was passing the Workload Domain rather than the identity source domain when assigning groups and users.
+- Fixed `Add-ProtectionGroup` cmdlet example wording.
```

Comment:
I think we can remove the notice for these fixes since both functions will be new in the v2.8.0 release.

Reply:
Fixed